lib/user.ts

```diff
@@ -45,7 +52,7 @@ export interface UserProps {
   /**
    * A user's lobbies
    */
-  readonly lobbies?: string[];
+  readonly lobbies?: LobbyMetadata[];
```

Comment:
this will completely change our implementation in the database.. are u sure you want to add this instead of doing smth like lobbyMetadata? if so, lets make it the same for the lobby class and just get rid of the id stuff?

Reply:
Oh yea i was thinking that this might be better long term cuz otherwise we would return both an array of ids and the list of metadata which includes the ids anyway?

Reply:
i like it! then maybe lets just refactor everything to be like this?

Reply:
ye sounds good